var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -309,36 +314,12 @@ def cmake_args(self):
                 '-DDEAL_II_WITH_ARPACK=OFF'
             ])
 
-        # Assimp
-        if '+assimp' in spec:
-            options.extend([
-                '-DDEAL_II_WITH_ASSIMP=ON',
-                '-DASSIMP_DIR=%s' % spec['assimp'].prefix
-            ])
-        else:
-            options.extend([
-                '-DDEAL_II_WITH_ASSIMP=OFF'
-            ])
-
         # since Netcdf is spread among two, need to do it by hand:
         if '+netcdf' in spec and '+mpi' in spec:
-            # take care of lib64 vs lib installed lib locations:
-            if os.path.isdir(spec['netcdf-cxx'].prefix.lib):
-                netcdfcxx_lib_dir = spec['netcdf-cxx'].prefix.lib
-            else:
-                netcdfcxx_lib_dir = spec['netcdf-cxx'].prefix.lib64
-            if os.path.isdir(spec['netcdf'].prefix.lib):
-                netcdf_lib_dir = spec['netcdf'].prefix.lib
-            else:
-                netcdf_lib_dir = spec['netcdf'].prefix.lib64
-
+            netcdf = spec['netcdf-cxx'].libs + spec['netcdf'].libs
             options.extend([
                 '-DNETCDF_FOUND=true',
-                '-DNETCDF_LIBRARIES=%s;%s' % (
-                    join_path(netcdfcxx_lib_dir,
-                              'libnetcdf_c++.%s' % dso_suffix),
-                    join_path(netcdf_lib_dir,
-                              'libnetcdf.%s' % dso_suffix)),
+                '-DNETCDF_LIBRARIES=%s' % netcdf.joined(';'),
                 '-DNETCDF_INCLUDE_DIRS=%s;%s' % (
```

Comment:
Can you do the same joined path for the includes as you did for the libs?

Reply:
I am not too familiar with “headers” interface, so I would leave it as-is

Reply:
Ok!